fastjet/internal/delaunay/point.go

```diff
@@ -20,10 +20,10 @@ type Point struct {
 	adjacentTriangles triangles // adjacentTriangles is a list of triangles containing the point.
 	nearest           *Point
 	dist2             float64 // dist2 is the squared distance to the nearest neighbor.
-	// id is used when points are removed. Copies of the points around the point to
-	// be removed are made. The ID is set incremental in counterclockwise order. It identifies
-	// the original. It is also used to determine whether a Triangle is inside or outside the
-	// polygon formed by all those points.
+	// id is a unique identifier, that is assigned incrementally to a point on insertion.
```

Comment:
BTW, could you also document in the doc-string of `Point` that one should use the `Equal` method of `Point` to test whether 2 points are equal ?
(and not use the `==` operator. the comment on `id` made me think of that. it's a similar use case (trap, really) to `time.Time`...)

Reply:
Done